src/frontend/components/router-tree/router-tree.ts

```diff
@@ -52,7 +52,7 @@ export class RouterTree {
     };
 
     const nodes = this.tree(d3.hierarchy(
-      root.children.length > 1 ? root : root.children[0], d => d.children));
+      root.children.length === 0 || root.children.length > 1 ? root : root.children[0], d => d.children));
```

Comment:
@rajinder-yadav The ternary expression is always passing an object, but it's either the `root` object, or the first of that object's `children` property (an array).